client/landing/stepper/declarative-flow/internals/steps-repository/launchpad/task-helper.ts

```diff
@@ -210,14 +210,15 @@ export function getEnhancedTasks(
 						actionDispatch: () => {
 							if ( site?.ID ) {
 								const { setPendingAction, setProgressTitle } = dispatch( ONBOARD_STORE );
-								const { launchSite } = dispatch( SITE_STORE );
+								const { launchSite, saveSiteSettings } = dispatch( SITE_STORE );
 
 								setPendingAction( async () => {
 									setProgressTitle( __( 'Launching Website' ) );
 									await launchSite( site.ID );
+									await saveSiteSettings( site.ID, { launchpad_screen: 'off' } );
```

Comment:
Adding `saveSiteSettings...` here will be redundant and affect all the flows.

We handle updating the `launchpad_screen` option to `off` on WPCOM (launchpad.php) to ensure that users only are only able to launch and update this option under certain conditions.

Reply:
👋 To clarify, is it okay to turn the option off from Calypso in a step that's only run for the Sensei flow? (When we are setting up other options like the site intent.) Or would that interfere with something? I'd prefer not spreading around setting the option and a flow that relies on it to different parts of the system since that adds a potential source of issue.

Reply:
As we are going to eventually drop the launchpad, I have removed the code that affects other flows. Any more changes to our launchpad screen are going to be handled as part of another PR 667560ba720104391e8ec8f3f903644020c8e66f